src/frontend/parser.mly

```diff
@@ -246,10 +258,22 @@ sized_basic_type:
     { grammar_logger "MATRIX_var_type" ; SizedType.SMatrix (e1, e2) }
 
 top_var_decl_no_assign:
-  | tvt=top_var_type id=decl_identifier d=option(dims) SEMICOLON
+  | tvt=top_var_type id=decl_identifier SEMICOLON
+    {
+      grammar_logger "top_var_decl_no_assign" ;
+      {stmt=
+         VarDecl {decl_type= Sized (fst tvt);
+                  transformation=  snd tvt;
+                  identifier= id;
```

Comment:
I was going to complain about the formatting (two spaces before `snd`, semicolons at the end of lines) but `ocamlformat` doesn't work on `.mly` files and the formatting is messed up a bit all over the file so whatever, this is good enough.

Reply:
Yep, fair. Also this is fixed in #561